README.md

```diff
@@ -52,19 +52,16 @@ This will install go-npm under to your package.json file. It will also create a
 
 `$ rm -r node_modules`
 
-**Add postinstall and preuninstall scripts**
+**Add preinstall script**
 Here is the magic: You ask to run `go-npm install` after it completes installing your package. This will pull down binaries from Github or Amazon S3 and install in NPM's `bin` directory. Binaries under bin directory are immediately available for use in your Terminal.
 
 Edit `package.json` file and add the following:
 ```
 {
-    "postinstall": "go-npm install",
-    "preuninstall": "go-npm uninstall",
+    "preinstall": "go-npm install"
```

Comment:
The binaries are linked by npm / yarn etc. on install, so the binary has to be present before the installation. `preinstall` achieves it, whereas `postinstall` is too late